tests/ui/pages/catalog/catalog_page.ts

```diff
@@ -0,0 +1,59 @@
+import Button from '../../elements/button';
+import BasePage from '../base-page';
+
+const SELECTORS = {
+  clearAll: `button:has-text('Clear All')`,
+  filterWithSelect: filterName => `.MuiGrid-root.css-19dbjmo:has-text('${filterName}')`,
+  filterWithInput: filterName =>
+    `.MuiGrid-root.MuiGrid-item.MuiGrid-grid-xs-12.css-1vkoyw5:has-text('${filterName}')`,
+  filterSelect: `[role="button"]`,
+  filterInput: `[type="text"]`,
+  searchBar: `[placeholder="Search"]`,
+  searchButton: `.sc-kDDrLX.gpLWbw > div > div > div > div > [type="button"]`,
+  listOfFilters: `[role="listbox"]`,
+    filterOption: `[role="option"]`,
+  itemString: name => `a:has-text('${name}')`,
+  tab: name => `[role="tab"]:has-text('${name}')`,
+    optionJobInTypeFilter: `[aria-activedescendant="filter-types-option-1]`,
+};
+export default class CatalogPage extends BasePage {
+  get clearAll() {
+    return new Button(this.page, SELECTORS.clearAll);
+  }
+
+  async openFilterWithSelect(nameOfFilter: string) {
+      await this.page.waitForLoadState("networkidle");
+    await this.page.click(
+      `${SELECTORS.filterWithSelect(nameOfFilter)}`);
+  }
+
+  async chooseOption(optionOfFilter: string) {
+    await this.page.click(`${SELECTORS.listOfFilters} >> ${SELECTORS.filterOption}:has-text('${optionOfFilter}')`);
+  }
+
+  async chooseOptionJobInTypeFilter() {
+      await this.page.click(`${SELECTORS.listOfFilters} >> ${SELECTORS.optionJobInTypeFilter}`);
```

Comment:
You can choose the first element like this:
```
await page.locator('[type=radio]:left-of(:text("Label 3"))').first().click();
```
Src: https://playwright.dev/docs/selectors